src/app/file-manager/file-modal/file-modal.component.scss

```diff
@@ -1,44 +0,0 @@
-.mat-dialog-content {
```

Comment:
If the file is empty does it still need to be present?

Reply:
I guess not, but these type of default created files may be useful in the future. But we could delete it.

Reply:
Since the file is now empty, I'm not sure it's useful to keep it around, even if it could be useful in the future.